### PR TITLE
Remove tooltip. Remove warning icon and reading room icon from search results. Remove tooltip title

### LINKED ISSUE #3885: Title attribute text is the same as text or alternative text.
### Steps to recreate
Visit any record page. For example go to [record 99125410673606421](https://catalog.princeton.edu/catalog/99125410673606421)
Evaluate the page using WAVE. 

### URL
https://catalog.princeton.edu/catalog/99125410673606421

### Issues
The title attribute value is used to provide advisory information. It typically appears when the users hovers the mouse over an element. The advisory information presented should not be identical to or very similar to the element text or alternative text.

### WCAG Conformance level (A, AA, AAA)
None

### User impact
Redundant title text can be confusing.

### Recommended fix
In most cases the title attribute can be removed, otherwise modify it to provide advisory, but not redundant information. Note that the title text may or may not be read by a screen reader and is typically inaccessible to sighted keyboard users
Remove the title attribute from the three forms. Ask a question, Suggest a correction, Report harmful language.

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### LINKED ISSUE #3906: Remove tooltip
### Steps to recreate
Go to [search results](https://catalog.princeton.edu/?f%5Bformat%5D%5B%5D=Book) 

- Use the mouse and hover over the author name.  A tooltip is triggered with some extra information including the words 'Search' or 'Browse' accordingly.
- Use the keyboard and tabkey on the author name. The tooltip is not triggered.
- Use the screenreader. The user will get the same information with the keyboard user but not with the sighted user. 
- Use the screenreader and point the mouse on the author browse. The screenreader will prompt the use to listen to a description. 

### Issues
[Understanding SC 2.1.1](https://www.w3.org/WAI/WCAG22/Understanding/keyboard)

Pointer actions should have a keyboard equivalent 
The tooltip is adding an aria-describedby which is useful for a screenreader user but when they point the mouse on the field. This is not useful for a screenreader user. 

### WCAG Conformance level (A, AA, AAA)
A

### User impact
All users dont have the same experinece and do not get the same information.

### Recommended fix
Remove the data tooltip and title

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### LINKED ISSUE #3907: Remove the tooltip in combination with the icon-warning icon-request-reading-room
### Steps to recreate
Go to [search results Mudd](https://catalog.princeton.edu/?f%5Blocation%5D%5B%5D=Mudd+Manuscript+Library) 

- Use the mouse and hover over the yellow triangle icon warning.  A tooltip is triggered with extra information 'Items at this location must be requested'.
- Use the keyboard and tabkey. The icon is not focusable. The same information will not be provided to the tab key user.
- Use the screenreader the icon has an aria-hidden attribute even though it provides information to the sighted users who can use a mouse. 
- When you hover over the icon the tooltip adds an aria-describedby which is useless since it is aria-hidden.

### Issues
[Understanding SC 2.1.1](https://www.w3.org/WAI/WCAG22/Understanding/keyboard)

Pointer actions should have a keyboard equivalent 
The tooltip is adding an aria-describedby in the icon span which is useless since it has an aria-hidden true.
Availability can be misleading for different users since the icon tooltip is only read by some users.

### WCAG Conformance level (A, AA, AAA)
A

### User impact
All users don't have the same experience and do not get the same information.

### Recommended fix
Remove the icon warning icon reading room

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes
part of #3878

### PR DESCRIPTION
**Pull request by christinach**
closes #3885
closes #3906
closes #3907
part of #3878

### CODE REVIEW

**Comment by christinach** [comment]
the branch is deployed on catalog-staging

File: spec/system/searching_spec.rb [hunk 1]
```diff
@@ -68,9 +68,9 @@
 
   # We don't have the on-site icon at the moment. It appears as available. We might add it back in the future.
   context 'with items which are from aeon locations' do
-    it 'renders an accessible warning icon for requesting an item in a reading room' do
+    it 'does not render a warning icon for requesting an item in a reading room' do
```
Comment (sandbergja) [inline spec/system/searching_spec.rb:71]: Do we still need this test?

File: spec/components/aeon_request_button_component_spec.rb [hunk 2]
```diff
@@ -16,8 +16,8 @@
   it "renders a link with the appropriate classes" do
     expect(subject.css('a').attribute('class').to_s).to eq('request btn btn-xs btn-primary')
   end
-  it 'renders the typical title tooltip' do
-    expect(subject.css('a').attribute('title').text).to eq('Request to view in Reading Room')
+  it 'does not render a tooltip' do
+    expect(subject.css('a').attribute('title')).to be_falsey
```
Comment (sandbergja) [inline spec/components/aeon_request_button_component_spec.rb:20]: 👍🏻

**Review by sandbergja — APPROVED** [review]
Thanks, @christinach !  It will be good to get all of these out of the catalog.  My only question is: do we want to get rid of some of the tests as well, rather than setting them to `not_to`?  Not a blocker! 🐬

File: spec/system/searching_spec.rb (hunk 1 above)
Comment (christinach) [reply spec/system/searching_spec.rb:71]: I can remove them. I kept them to make sure the icon is not there anymore. But it can be deleted now

**Pull request merged**